utils/utils.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"io"
+	"os"
+)
+
+func ReadFileAll(filepath string) ([]byte, error) {
```

Comment:
You can use `os.ReadFile`

Reply:
There is another concern I have here, do we need to verify the file size? For example, not reading it if it exceeds a certain threshold.